pyproject.toml

```diff
@@ -36,7 +36,7 @@ known_third_party = "mysql.connector"
 [tool.flake8]
 max-line-length = 99
 max-doc-length = 99
-max-complexity = 12
+max-complexity = 13
```

Comment:
im not sure i am completely onboard with this change. we should try using sensible defaults here

Reply:
The default on the charm template is 10, so this has already crept up a bit from the norm